app_dart/README.md

```diff
@@ -80,13 +85,30 @@ To update the cron tasks in the App Engine project, run:
 $ gcloud app deploy cron.yaml
 ```
 
-### ssh into instance
+### Local development
+
+#### Using physical machine
+
+* Starting server
+
+```sh
+export COCOON_USE_IN_MEMORY_CACHE=true
```

Comment:
This should be a part of setting up the environment.

Reply:
Added a set up environment bullet.